synthtool/__main__.py

```diff
@@ -70,8 +70,8 @@ def main(synthfile: str, metadata: str, extra_args: Sequence[str]):
     if os.path.lexists(synth_file):
         synthtool.log.debug(f"Executing {synth_file}.")
         # https://docs.python.org/3/library/importlib.html#importing-a-source-file-directly
-        spec = importlib.util.spec_from_file_location("synth", synth_file)
-        synth_module = importlib.util.module_from_spec(spec)
+        spec = util.spec_from_file_location("synth", synth_file)
```

Comment:
I was receiving linting errors locally without switching to this approach (upstream regression perhaps?).

Reply:
Honestly that should be a linting error. It's not always guaranteed that submodules are actually loaded when you import a top-level module.

That said, I'd prefer that you instead change the above import to `import importlib.util` instead. `util` is too common of a name to be meaningful.